core/integration/bookkeeping/plugin.go

```diff
@@ -431,6 +431,24 @@ func (p *Plugin) CallStack(data interface{}) (stack map[string]interface{}) {
 		rnString := strconv.FormatUint(uint64(runNumber), 10)
 
 		flps := env.GetFLPs()
+
+		// The enabled status of CTP Readout and its FLP is governed by the variable ctp_readout_enabled, so it requires
+		// special treatment to be included in the list of FLPs
+		ctpReadoutEnabled, err := strconv.ParseBool(env.GetKV("", "ctp_readout_enabled"))
+		if err != nil {
+			log.WithError(err).
+				WithField("run", runNumber64).
+				WithField("partition", envId).
+				WithField("call", "StartOfRun").
+				Warning("cannot parse CTP readout enabled")
```

Comment:
```suggestion
				Warning("cannot parse ctp_readout_enabled")
```
For clarity, I would use the exact variable name, so users are not mislead that they are using a wrong variable name.